Specs/DateTime/French/DateTimeModel.json

```diff
@@ -20902,5 +20902,53 @@
         "End": 46
       }
     ]
+  },
+  {
+    "Input": "Nous avons terminé notre cours il y a 30 minutes.",
+    "Context": {
+      "ReferenceDateTime": "2021-05-20T15:30:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "il y a 30 minutes",
+        "Start": 31,
+        "End": 47,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2021-05-20T15:00:00",
+              "type": "datetime",
+              "value": "2021-05-20 15:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Je l'ai commandé il y a 24 heures.",
```

Comment:
Can "il ya" appear in informal French? If so, we should support it if it doesn't cause undesired side effects.
Should we also have more specs with depuis?
And lastly, can you this spec? "Il y a à peu près un mois".
If not supported and too complex to address, add this new spec as disabled and open an issue to fix it. Thanks.

Reply:
test cases added.
"Il y a à peu près un mois" is now parsed as date but the modifier "à peu près" is ignored. 
The problem is that Merged looks for modifiers only at the beginning/end of the extraction, so a larger modification is required to target modifiers within the extraction.

Reply:
Got it. Let's open an issue for it, but this is lower priority for now.